src/Microsoft.TemplateSearch.Common/Abstractions/ITemplatePackageInfo.cs

```diff
@@ -23,5 +25,15 @@ public interface ITemplatePackageInfo
         /// Optional, might be 0 in case search provider cannot provide number of downloads.
         /// </summary>
         public long TotalDownloads { get; }
+
+        /// <summary>
+        /// Gets the list of template package owners.
+        /// </summary>
+        public IReadOnlyList<string> Owners { get; }
+
+        /// <summary>
+        /// Gets the indication if the package is verified (if supported by the source).
```

Comment:
Maybe we can make it more clear in the comments: what exactly is verified? Is it the package content or the owner? (I believe in our case, it is the owner, right? It is verified that the owner is who he says he is.)

Reply:
"Verified" in this context indicates that the package ID is under a reserved namespace.
https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation

Reply:
since it is an abstraction, I would say it's up to actual implementation to define the meaning. I will add a remark for NuGet.org with link above.